fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/ExpressionConverter.java

```diff
@@ -17,18 +17,169 @@
+                visit(greaterThan.child(0), context),
+                visit(greaterThan.child(1), context));
+    }
+
+    @Override
+    public Expr visitGreaterThanEqual(GreaterThanEqual greaterThanEqual, PlanContext context) {
+        return new BinaryPredicate(Operator.GE,
+                visit(greaterThanEqual.child(0), context),
+                visit(greaterThanEqual.child(1), context));
+    }
+
+    @Override
+    public Expr visitLessThan(LessThan lessThan, PlanContext context) {
+        return new BinaryPredicate(Operator.LT,
+                visit(lessThan.child(0), context),
+                visit(lessThan.child(1), context));
+    }
+
+    @Override
+    public Expr visitLessThanEqual(LessThanEqual lessThanEqual, PlanContext context) {
+        return new BinaryPredicate(Operator.LE,
+                visit(lessThanEqual.child(0), context),
+                visit(lessThanEqual.child(1), context));
     }
+
+    @Override
+    public Expr visitNot(Not not, PlanContext context) {
+        return new org.apache.doris.analysis.CompoundPredicate(
+                org.apache.doris.analysis.CompoundPredicate.Operator.NOT,
+                visit(not.child(0), context),
+                null);
+    }
+
+    @Override
+    public Expr visitNullSafeEqual(NullSafeEqual nullSafeEqual, PlanContext context) {
+        return new BinaryPredicate(Operator.EQ_FOR_NULL,
+                visit(nullSafeEqual.child(0), context),
+                visit(nullSafeEqual.child(1), context));
+    }
+
+    /**
+     * Convert to stale literal.
+     */
+    @Override
+    public Expr visitLiteral(Literal literal, PlanContext context) {
+        DataType dataType = literal.getDataType();
+        if (dataType instanceof BooleanType) {
+            return new BoolLiteral((Boolean) literal.getValue());
+        } else if (dataType instanceof DoubleType) {
+            return new FloatLiteral((Double) literal.getValue(), Type.DOUBLE);
+        } else if (dataType instanceof IntegerType) {
+            return new IntLiteral((Long) literal.getValue());
+        } else if (dataType instanceof NullType) {
+            return new NullLiteral();
+        } else if (dataType instanceof StringType) {
+            return new StringLiteral((String) literal.getValue());
+        }
+        throw new RuntimeException(String.format("Unsupported data type: %s", dataType.toString()));
+    }
+
+    @Override
+    public Expr visitFunctionCall(FunctionCall function, PlanContext context) {
+        List<Expr> paramList = new ArrayList<>();
+        for (Expression expr : function.getFnParams().getExpression()) {
+            paramList.add(visit(expr, context));
+        }
+        return new FunctionCallExpr(function.getFnName().toString(), paramList);
+    }
+
+    @Override
+    public Expr visitBetweenPredicate(BetweenPredicate betweenPredicate, PlanContext context) {
+        return new org.apache.doris.analysis.BetweenPredicate(
```

Comment:
@EmmyMiao87 could be execute between predicate expr? if be cannot do that, we should throw exception if there have a BetweenPredicate